### ISSUE TITLE
Effects plugin: Parameter labels in automation and accessibility are generic compared to full Surge

### ISSUE THREAD

**Issue by pitermach** [surge-synthesizer/surge#6052]
I couldn't think of a good issue title, but hopefully the reproduction steps will explain what I mean.

First, add an effect to any slot in Surge XT, IE a reverb 2 into Scene A's first slot which is what the UI starts in. If you now have a screen reader speak the first parameter slider or look at it in an accessibility inspector it's labeled as "FXA1 pre delay" and its value is reported in seconds. Similarly, if you look through the list of automatable parameters in the host it has a similar label and also reports real world values in seconds.

Now load up an instance of Surge XT effects and select Reverb 2 in that. Here, the slider for the first parameter for accessibility is labeled as "parameter 0 knob", while its value is just reported as a number between 0 and 1. The actual name and value do also appear in the UI, but as a separate element that you need to keep checking after you adjust something. The "tempo match" and "deactivate" toggles that some parameters get have similarly generic labels. 

Same goes for automation, where the parameters just appear as "FX parameter 0, FX parameter 1..." and don't expose real world values to the host.

I opened this in a separate issue rather than commenting in the accessibility one because of the automation aspect, since the cryptic names would affect anyone deciding to automate the effects plugin.

**Comment by baconpaul** [comment]
Oh yeah this is just sloppy. The should-rename code which flushes automation labels isn’t in the fx unit. I should fix that for sure thank you

**Comment by baconpaul** [comment]
OK @pitermach I just went and fixed these all up in the FX. Tested with both acc inspector and voiceover on macOS and it seemed way better (proper names proper units, invalidate when values change, etc...)

Next nightly should have it. I'll close this issue when we merge but if there's still something missing please do reopen it.

**Comment by pitermach** [comment]
Thanks for looking at this. Just had a moment to try out the changes, which work great. I found a slight bug though. 

The labels only appear after you change the effect. If you look at the UI when the plugin is first loaded or you close the plugin window and open it again the parameters will revert to the generic names.

Also, could you apply a similar fix to the way the params are exposed for host automation? They have similar vague names and values there.

**Comment by baconpaul** [comment]
Ahh yes I need to do that at the outset. Reopening.

Will look into vst params. That's a bit trickier.

**Comment by baconpaul** [comment]
OK the VST3 labels are harder. I opened #6090 for that but will do a push today for the label on startup bug.
